### PR TITLE
[opt](index) add more inverted index query profile

### PR DESCRIPTION
**Pull request by xiaokang**
add more inverted index query profile:
- InvertedIndexQueryNullBitmapTime
- InvertedIndexQueryFileExistsTime
- InvertedIndexSearcherCacheHit
- InvertedIndexSearcherCacheMiss

### CODE REVIEW

**Comment by xiaokang** [comment]
run buildall

File: be/src/olap/rowset/segment_v2/inverted_index_reader.cpp [hunk 1]
```diff
@@ -100,6 +100,7 @@ bool InvertedIndexReader::_is_match_query(InvertedIndexQueryType query_type) {
 }
 
 bool InvertedIndexReader::indexExists(io::Path& index_file_path) {
+    // SCOPED_RAW_TIMER(&stats->inverted_index_query_file_exists_timer);
```
Comment (zzzxl1993) [inline be/src/olap/rowset/segment_v2/inverted_index_reader.cpp:103]: can delete it

**Review by zzzxl1993 — APPROVED** [review]
LGTM

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

**Comment by xiaokang** [comment]
run buildall

**Pull request merged**